docusaurus.config.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
```

Comment:
please move this file and `static` folder to `docs`.
We can build it using `docusaurus start docs`

Reply:
We are building using `docusaurus` commands

> Every website needs assets: images, stylesheets, favicons etc. In such cases, you can create a directory named static at the root of your project.
https://docusaurus.io/docs/static-assets

Reply:
![image](https://user-images.githubusercontent.com/8336157/143016839-0c7ab8b8-e1ce-4a3c-9a91-f105fdbdc1c5.png)

We can use this one and achieve the same isolation that you wanted initially without having nested `node_modules`.
Ping me here if you don't have time for this change and I can do it myself to push this PR over the finish line.